doc/project/license.rst

```diff
@@ -95,6 +95,13 @@ Colormaps and themes
    .. literalinclude:: ../../LICENSE/LICENSE_YORICK
       :language: none
 
+.. dropdown:: Turbo
+   :class-container: sdd
+
+   The Turbo colormap was contributed directly to Matplotlib by its author.
```

Comment:
```suggestion
   The Turbo colormap was contributed directly to Matplotlib by its author in :ghpull:`15275`
```
anybody who needs this information is likely going to need the pull request for verification